src/components/picker/picker.tsx

```diff
@@ -203,6 +203,8 @@ export class Picker {
     @Watch('value')
     protected onChangeValue(newValue = [], oldValue = []) {
         this.chips = this.createChips(this.value);
+        newValue = newValue || [];
```

Comment:
newValue = [] and oldValue = [] by default. Right? I think we don't have to set it again.

Reply:
Unfortunately not. If `null` is provided as an argument then `newValue` will be `null` and not `[]`